src/app/webserver/central-certificates/central-certificate.component.ts

```diff
@@ -23,10 +23,8 @@ import { CentralCertificateConfiguration } from './central-certificates';
         <div *ngIf="_configuration">
             <fieldset class="path">
                 <label>Physical Path</label>
-                <button title="Select Folder" [class.background-active]="fileSelector.isOpen()" class="right select" (click)="fileSelector.toggle()" [attr.disabled]="isPending || null" ></button>
-                <div class="fill">
```

Comment:
What's the purpose of removing this class

Reply:
Previously, the fill class was used to fill the space between the input text box and the file selector button becuase the file selector button was placed before the edit text box.

Now, that was fixed with switching the order and the button is supposed to be put right after the edit box automatically. 

The fill class can't be used anymore because the edit box will occupy whole line which will make the button is displayed in the next line. That's why I had to remove the fill class.